examples/headland_inversion/README.md

```diff
@@ -0,0 +1,210 @@
+# Headland Channel - Friction Field Calibration
+
+## Forward run
+
+The forward run is not included here, instead, only the time series `.hdf5` files at each station are provided. 
```

Comment:
Fair enough, added it now. I had intentionally done this because in reality you only get the ground truth data (i.e. no forward run!) - so it was just trying to emulate that. Probably best to include it as you suggest.